shared/torngit/github.py

```diff
@@ -135,22 +151,167 @@ def prepare(self, query_name: str, variables: dict) -> Optional[dict]:
 class Github(TorngitBaseAdapter):
     service = "github"
     graphql = GitHubGraphQLQueries()
-    urls = dict(
```

Comment:
Took me a sec to figure out, but we're not using these altogether it seems?

Reply:
yep! Turns out only 1 of those was in use, the other endpoints we use are written out in each function. The refactor I'm going to do is going to include something to replace this - a collection of the urls we use